Automattic/jetpack
# New Disconnect Dialog

#### Changes proposed in this Pull Request:

* Adds new, redesigned disconnect dialog
* Disconnect dialog now dynamically pulls in usage stats of Jetpack on the site
* Prep work for Survey Step in disconnect dialog
* Prep work for Plugins Install mobile

**Two Site Benefits - Mobile**
<img width="325" alt="Screen Shot 2019-09-24 at 3 39 04 PM" src=https://user-images.githubusercontent.com/2810519/65555872-a0026f00-dee2-11e9-9650-b1d836a47f3e.png>

**Three Site Benefits - Mobile**
<img width="325" alt="Screen Shot 2019-09-24 at 3 39 04 PM" src=https://user-images.githubusercontent.com/2810519/65555877-a2fd5f80-dee2-11e9-9ae2-71506d8f7e87.png>

**Single Site Benefit - Wide**
<img width="325" alt="Single Site Benefit - Wide" src="https://user-images.githubusercontent.com/2810519/65555705-31251600-dee2-11e9-8ca2-c0396f819e3c.png">

**Two Site Benefits - Wide**
<img width="325" alt="Two Site Benefits - Wide" src="https://user-images.githubusercontent.com/2810519/65555696-2cf8f880-dee2-11e9-9423-6079ada7d14a.png">

**Three Site Benefits - Wide**
<img width="325" alt="Three Site Benefits - Wide" src="https://user-images.githubusercontent.com/2810519/65555692-2b2f3500-dee2-11e9-87c2-4de2e4b2c565.png">

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Internal reference: parBdM-hs-p2

Redesign of Jetpack Disconnect Dialog 

#### Testing instructions:






1. Navigate to `/wp-admin/admin.php?page=jetpack#/dashboard`
2. Click "Manage Site Connection"
<img width="403" alt="Screen Shot 2019-09-24 at 3 53 46 PM" src="https://user-images.githubusercontent.com/2810519/65556153-84e42f00-dee3-11e9-98b4-cf02e6c9c7b0.png">
3. Verify that your dialog roughly matches one of the above screenshot.
4. Modify the following code snippet and insert into `_inc/client/components/jetpack-termination-dialog/features.jsx` before the line `const siteBenefitCount = siteBenefits.length;` with varying amounts of additional site benefits to test the various cases.

```js
	siteBenefits = siteBenefits.concat( [
		{
			title: 'Brute Force Protection',
			description: 'The number of malicious login attempts blocked by Jetpack.',
			amount: 32030,
			gridIcon: 'lock',
		},
		{
			title: 'Contact Forms',
			description: 'The number of live Jetpack forms on your site right now.',
			amount: 31,
			gridIcon: 'align-image-center',
		},
		{
			title: 'Publicize',
			description: 'The number of live social media connections, powered by Jetpack.',
			amount: 3,
			gridIcon: 'share',
		},
		{
			title: 'Subscribers',
			description: 'The number of people subscribed to your updates through Jetpack.',
			amount: 4200,
			gridIcon: 'user',
		},
	] );
```


#### Proposed changelog entry for your changes:


Redesign of Jetpack Disconnect Dialog
